Exchange term 'user' with more descriptive words (#1330)
shopware/docs: guides/plugins/apps/payment.md

## Prepared payments

> With Shopware 6.4.9.0, you can use prepared payments to enhance your checkout process beyond forwarding to a payment provider.

Reviewer: ```suggestion
With Shopware `6.4.9.0`, you can use prepared payments to enhance your checkout process beyond forwarding to a payment provider.
```

</Tabs>

::: warning
Keep in mind that if the integration into the checkout process does not work as expected,
your customer might not be able to use the prepared payment.
This is especially valid for after order payments, since there the order already exists.
For these cases, you should still offer a traditional synchronous / asynchronous payment flow.
Don't worry, if you have set the transaction state in your capture call to anything but open, the asynchronous payment process will not be started immediately after the prepared payment flow.
:::

## Refund

> With Shopware 6.4.12.0, we have also added basic functionality to be able to refund payments.

Reviewer: ```suggestion
With Shopware `6.4.12.0`, we have also added basic functionality to be able to refund payments.
```

Recurring orders and payments require the Subscriptions feature, available exclusively in our [paid plans](https://www.shopware.com/en/pricing/).
:::

Recurring payments are a special case of payment that is used for handling recurring orders, such as subscriptions.
The request and response payloads are similar to the synchronous payment flow.
At this point, a valid running billing agreement between the customer and the PSP should exist.
Use any of the other payment flows to capture the initial order and create such an agreement during the checkout.

> Afterwards, the payment can be captured via this flow for every recurring payment order.

Reviewer: ```suggestion
Afterward, this flow can capture the payment for every recurring payment order.
```
